src/main/java/net/dv8tion/jda/internal/interactions/CommandDataImpl.java

```diff
@@ -243,18 +244,17 @@ public CommandDataImpl addSubcommandGroups(@Nonnull SubcommandGroupData... group
         if (groups.length == 0)
             return this;
         checkType(Command.Type.SLASH, "add subcommand groups");
-        if (!allowGroups)
+        if (!allowSubcommands)
```

Comment:
was something changed to allow groups and subcommands to be added at the same time? I'm confused as to why this changed

Reply:
Yes, according to the [docs](https://discord.com/developers/docs/interactions/application-commands#subcommands-and-subcommand-groups) you can mix subcommands and groups.

```
VALID

command
|
|__ subcommand-group
    |
    |__ subcommand
|
|__ subcommand
```

Reply:
Interesting! We will need to update our documentation then
Currently, variations of this are in the codebase: `Also throws if you try to mix subcommands/options/groups in one command.`